src/FNA3D_Driver_Vulkan.c

```diff
@@ -2094,6 +2102,12 @@ static uint8_t VULKAN_INTERNAL_FindMemoryType(
 			(memoryProperties.memoryTypes[i].propertyFlags & properties) == properties	)
 		{
 			*result = i;
+			FNA3D_LogInfo(
+				"Found suitable memory type; type index: %u, type filter: %u, properties: %u",
```

Comment:
```suggestion
				"Found suitable memory type; type index: %u, type filter: %X, properties: %X",
```

I also ended up changing the LogError below to this:

```
FNA3D_LogWarn("Failed to find memory properties %X, filter %X\n", properties, typeFilter);
```